axual-webclient/topic_configs.go

```diff
@@ -65,3 +65,38 @@ func (c *Client) DeleteTopicConfig(id string) error {
 	}
 	return nil
 }
+
+func (c *Client) GetTopicConfigPermissions(topicConfigID string, permType string) ([]PermissionResponse, error) {
+	var perms []PermissionResponse
+	err := c.RequestAndMap("GET", fmt.Sprintf("%s/stream_configs/%s/permissions?type=%s", c.ApiURL, topicConfigID, permType), nil, nil, &perms)
+	if err != nil {
+		return nil, err
+	}
+	return perms, nil
+}
+
+func (c *Client) DeleteTopicConfigPermissions(topicConfigID string, request PermissionRequest) error {
+	marshal, err := json.Marshal(request)
+	if err != nil {
+		return err
+	}
+	headers := map[string]string{"Content-Type": "application/json"}
+	err = c.RequestAndMap("DELETE", fmt.Sprintf("%s/stream_configs/%s/permissions?type=browse", c.ApiURL, topicConfigID), strings.NewReader(string(marshal)), headers, nil)
```

Comment:
Sorry, not going to do this because there are 63 queries to endpoints, all formatted the same way. I am not going to change this one for consistency. Also one could argue this is too verbose.

Reply:
Make sense, I was also reluctant to add the comments. Let's keep it consistent then